aea/multiplexer.py

```diff
@@ -73,6 +73,15 @@ def __init__(
 
         self.set_loop(loop if loop is not None else asyncio.new_event_loop())
 
+    @property
+    def default_connection(self) -> Connection:
+        """Get the default connection."""
+        if self._default_connection is not None:
+            return self._default_connection
+        else:
+            # return the first
+            return self._connections[0]
```

Comment:
Would it not be better to by default set the default connection. It is a bit confusing otherwise..